### ISSUE TITLE
Тooltip is not removed when the value is empty

### ISSUE THREAD

**Issue by ixxvivxxi** [altrim/ember-cli-tooltipster#37]
Example:
component.js:
```
  tooltipContent: Ember.computed(
    'title',
    function () {
      if (this.get('title')) {
        return this.get('title');
      }
      return '';
    }
  ),
```
template.hbs
```
{{#tool-tipster content=tooltipContent}}
  item
{{/tool-tipster}}
```
step 1: title is not empty and tooltip shows title value
step 2: set the title empty value. The tooltip will remain old value, and it will be shown

**Comment by ixxvivxxi** [comment]
My solution:
```
  _onContentDidChange: observer('content', 'title', function () {
    run.scheduleOnce('afterRender', this, () => {
      let content = this.get('content') || this.get('title') || null;
      if (isHTMLSafe(content)) {
        content = content.toString();
      }
      if (this.get('tooltipsterInstance') !== null) {
        this.get('tooltipsterInstance').content(content);
      }
    });
  }),
```

**Comment by altrim** [comment]
@ixxvivxxi Just released `v0.5.4`, with your fix. Thanks for the help 👍
